lib/rubocop/cop/rspec/spec_file_path_format.rb

```diff
@@ -32,6 +32,11 @@ module RSpec
       #   # good
       #   whatever_spec.rb         # describe MyClass, type: :routing do; end
       #
+      # @example `IgnoreMetadata: {type=>[routing,models]}` (default)
+      #   # good
+      #   whatever_spec.rb         # describe MyClass, type: :routing do; end
+      #   whatever_spec.rb         # describe MyClass, type: :models do; end
```

Comment:
I propose:
```yaml
IgnoreMetadata:
  - prepare # ignore by presence of a key
  - type: model # ignore one value
  - type: routing # and another, too
  - skip: [database] # ignore when the value is an array of specific values
```

Luckily enough, YAML allows to mix all that, and to cover all ignored cases unambiguously.